Yona/Yona/Welcome/WelcomeViewController.swift

```diff
@@ -118,4 +81,32 @@ class WelcomeViewController: BaseViewController {
         }
     }
     
+    fileprivate func processEnvironmentURL(newEnvironmentURL:String, currentEnvironmentURL:String) {
+        EnvironmentManager.updateEnvironment(withURLString: newEnvironmentURL)
+        self.didEditTextField = false
+        if currentEnvironmentURL != newEnvironmentURL{
+            ActivitiesRequestManager.sharedInstance.getActivityDefaultCategories(onCompletion: { (success, message, code, activitiesReturned, error) in
+                if success { EnvironmentManager.updateEnvironment(withURLString: newEnvironmentURL)
```

Comment:
See my earlier comment about the code formatter. The if and else bodies should be on separate lines.

Reply:
Done